app/js/actions/members.js

```diff
@@ -10,6 +10,32 @@ function getMembersSuccess(members) {
   };
 }
 
+function saveAs(contents, filename) {
+  const element = document.createElement('a');
+  element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(contents));
+  element.setAttribute('download', filename);
+  element.style.display = 'none';
+  document.body.appendChild(element);
+  element.click();
+  document.body.removeChild(element);
+}
+
+export function downloadMembers() {
+  api.Memberships
+    .all({ active: new Date() }, true)
```

Comment:
I know the download attribute doesn't work in safari (unless they've added support recently) But otherwise yes.